docs/quickstart-exit.md

```diff
@@ -0,0 +1,91 @@
+# Clean up
+
+In this document you'll learn how to:
+
+* Exit the MySQL command client shell and the Docker container
+
+* Remove the Docker container and the Docker image
+
+* Remove the Docker volume.
+
+The steps are as follows:
+{.power-number}
+
+1. To exit the MySQL command client shell we use `exit`. You can also use the `\q` or `quit` commands. The execution of the statement also closes the connection.
+
+    Exit the MySQL command client shell and close the connection:
+
+    ```{.bash data-prompt="mysql>"}
+    mysql> exit
+    ```
+
+    ??? example "Expected output"
+
+        ```{.text .no-copy}
+        Bye
+        ```
+
+2. Remove docker containers and images, if you no longer need docker containers / images or want to free up disk space. To remove a docker container, use the command `docker rm` followed by the container name or the container ID. To remove a docker image, use the command `docker rmi` followed by the image ID or name and the tag.
+
+    * Remove `psmysql2` and `pxb` Docker containers:
+
+        ```{.bash data-prompt="$"}
+        $ docker container rm psmysql2 -f
+        ```
+
+        ??? example "Expected output"
+
+            ```{.text .no-copy}
+            psmysql2
+            ```
+
+        
+        ```{.bash data-prompt="$"}
+        $ docker container rm pxb -f
+        ```
+
+        ??? example "Expected output"
+
+            ```{.text .no-copy}
+            pxb
+            ```
+
+    * Remove `percona/percona-xtrabackup:8.0.34` Docker image
```

Comment:
Do i need to remove the psmysql image too?

Reply:
added